shopware/docker-dev/0.1/manifest.json

```diff
@@ -11,12 +11,10 @@
                 "    - 9998:9998",
                 "    - 5173:5173",
                 "    - 5773:5773",
+                "  env_file:",
+                "    - .env.local",
                 "  environment:",
                 "    APP_ENV: ${APP_ENV-dev}",
-                "    HOST: 0.0.0.0",
-                "    APP_URL: http://localhost:8000",
-                "    DATABASE_URL: mysql://root:root@database/shopware",
-                "    MAILER_DSN: smtp://mailer:1025",
```

Comment:
but those values are not in the .env.local file 🤔 . when I create a new project

Reply:
Hmm replacing seems not possible. I have readded almost all of them.